app/helpers/word-pluralize.js

```diff
@@ -0,0 +1,12 @@
+import {helper} from 'ember-helper';
+
+export function wordPluralize(params) {
```

Comment:
Ember Data brings [Ember Inflector](https://emberjs.com/api/data/classes/Ember.Inflector.html) in as a dependency so we can use proper inflection to pluralize words, it also extends the Ember.String prototype so you can do something like...

```
import Ember from 'ember';
const {String: {pluralize}} = Ember;

pluralize(word);
```

It also gives us a [`{{pluralize}}`](https://emberjs.com/api/data/classes/DS.Ember.HTMLBars.helpers.html#method_pluralize) helper that is used elsewhere in the app.